sklearn/dummy.py

```diff
@@ -107,7 +107,8 @@ def fit(self, X, y, sample_weight=None):
             Returns self.
         """
         X = check_array(X, accept_sparse=['csr', 'csc', 'coo'],
-                        force_all_finite=False)
+                        force_all_finite=False, dtype=None,
+                        ensure_min_features=0)
```

Comment:
Is there any specific reason adding ensure_min_features=0? No other place in current codebase do so. It's a classifier/regressor so I suppose we should at least feed it with some features?

Reply:
The proposal in #9832 suggested than we make make the input validation less strict by adding ensure_min_features=0. However, I agree with you that since it is a classifier/regressor we should feed it some features.

Reply:
OK, seems that jnothman and amueller don't oppose. I won't oppose although it seems a bit strange. Please add similar test for a zero feature array. I think my LGTM is close :)

Reply:
I'm not sure what a zero feature array is. Is it an array of null values or an array of zeros?

Reply:
Something like `np.array([[],[],[]])` (this is why I think it's strange.). It's always good to make sure that your test fail on master and pass in the PR.